layouts/partials/function/resource.html

```diff
@@ -1,13 +1,14 @@
 {{- $resource := 0 -}}
 {{- $url := urls.Parse .Path -}}
 {{- if not (partial "function/isUrlRemote.html" $url) -}}
+    {{- $dest := strings.TrimPrefix "./"  $url.Path -}
```

Comment:
parse failed: template: partials/function/resource.html:4: illegal number syntax: "-"

should close with `-}}`

Reply:
fixed